delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -683,7 +539,6 @@ def cleanupS3BucketPolicy(service, domain, stage, bucket) {
 		} catch(ex) {
             resetCredentials()
 			if(ex.getMessage().indexOf("groovy.json.internal.LazyMap") < 0) {
-				updateServiceInDB(var_db_service_id, configLoader.AWS.API.PROD_ID, configLoader.AWS.REGION, "deletion_failed");
```

Comment:
we should send an event here with status failed.

Reply:
Modified the code for sending failed event